snuba/clickhouse.py

```diff
@@ -375,6 +376,21 @@ def for_schema(self):
         return ', '.join(column.for_schema() for column in self.columns)
 
 
+class TableSchema(object):
+    """
+    Represents the full set of columns in a clickhouse table, this only contains
+    basic metadata for now. The code to generate the schema comes in a followup PR.
+    """
+
+    def __init__(self):
+        self.CLICKHOUSE_CLUSTER = None
```

Comment:
I think this may be a design issue. CLICKHOUSE_CLUSTER is stored in settings today and it is overridden in prod, thus it is not visible here. Whether we can move it here as an attribute of the dataset is a separate question since none is actually using that value today because it is only accessed by get_dist_table_definition function which is not called by anyone.
Will think over it whether we can just remove it for now. I do not have an answer yet.

Reply:
reverting to the old way of setting the cluster. There is no strong reason to move the cluster name into the dataset for now and it will be easy to move later.